src/components/projects/Entry.jsx

```diff
@@ -12,11 +13,22 @@ import "./Entry.scss";
 
 function Entry() {
   const [memberList, setMemberList] = useState([]);
+  const [fileSelected, setFileSelected] = useState(null);
   const dispatch = useDispatch();
   const history = useHistory();
   const search = useLocation().search;
   const idProject = new URLSearchParams(search).get("id");
 
+  const onFileChange = (event) => {
+    setFileSelected(event.target.files[0]);
+  };
+
+  const onFileUpload = async () => {
+    const fileName = await uploadFileToBlob(fileSelected);
+    setFileSelected(null);
+    return fileName;
+  };
```

Comment:
no error handling for this guy?

Reply:
should see if the result doesn't have an error, if there's an error then show alert to let the user know about the error.

Reply:
done